### ISSUE TITLE
These Op Not Support: Tensorflow::FusedBatchNormV3

### ISSUE THREAD

**Issue by xinyidaren** [alibaba/MNN#538]
环境：tensorflow2.0
代码如下：
import tensorflow as tf
from tensorflow import keras

input1 = keras.Input(shape=(28,28,1),dtype=tf.float32)
_x = keras.layers.Conv2D(64,(3,3),strides = 2,activation='relu',use_bias = False)(input1)
output = tf.keras.layers.BatchNormalization()(_x, training=False)
model = keras.Model(inputs=[input1],outputs=[output])

model.save('mnist_cnn.h5')

模型导出，使用tensorflow1.15环境下
代码如下:

from tensorflow.keras.models import load_model
import tensorflow as tf
import os
import os.path as osp
from tensorflow.keras import backend as K
#输入路径参数
input_path = '.'
#输出路径参数
output_dir = '.'
weight_file = 'mnist_cnn.h5'
weight_file_path = osp.join(input_path, weight_file)
model = load_model(weight_file_path)
output_graph_name = weight_file[:-3] + '.pb'
#转换函数
def freeze_session(session, keep_var_names=None, output_names=None, clear_devices=True):

    from tensorflow.python.framework.graph_util import convert_variables_to_constants
    graph = session.graph
    with graph.as_default():
        freeze_var_names = list(set(v.op.name for v in tf.global_variables()).difference(keep_var_names or []))
        output_names = output_names or []
        output_names += [v.op.name for v in tf.global_variables()]
        # Graph -> GraphDef ProtoBuf
        input_graph_def = graph.as_graph_def()
        if clear_devices:
            for node in input_graph_def.node:
                node.device = ""
        frozen_graph = convert_variables_to_constants(session, input_graph_def,
                                                      output_names, freeze_var_names)
        return frozen_graph

frozen_graph = freeze_session(K.get_session(),
                              output_names=[out.op.name for out in model.outputs])

tf.train.write_graph(frozen_graph, "model", output_graph_name, as_text=False)
mnn0.2.1.5转换时最终报错：These Op Not Support: Tensorflow::FusedBatchNormV3

请问是我的某一步的操作不对吗？training=False已做，但还是报错

**Comment by czy2014hust** [comment]
[在此](https://github.com/alibaba/MNN/blob/62c5ca47964407508a5fa802582e648fc75eb0d9/tools/converter/source/optimizer/tfextra/TFBatchNormalMerge.cpp#L104)添加一行
```c++
TFExtraManager::get()->insert("FusedBatchNormV3", std::shared_ptr<TFExtraManager::Transform>(new BatchNormalTransform));

**Comment by wangpupanjing** [comment]
我看代码里已经添加 了啊，但还是报这个错误
